lib/cdo/script_constants.rb

```diff
@@ -297,6 +297,8 @@ module ScriptConstants
     HELLO_WORLD_ANIMALS = 'hello-world-animals'.freeze,
     HELLO_WORLD_EMOJI = 'hello-world-emoji'.freeze,
     HELLO_WORLD_RETRO = 'hello-world-retro'.freeze,
+    HELLO_WORLD_SPACE = 'hello-world-space'.freeze,
```

Comment:
oh actually, I think these should be in the `hoc` section of this list with `hello-world-food-2021` and the other units from last hoc

Reply:
Sorry -- the unit names from last year kept changing and we kept adding "temporary" scripts to this file. Looks like we missed cleaning a few of these up :(

Reply:
Updated! I looked at removing the 'hello-world-retro'/emoji/animals/food, but it looks like those scripts do exist separately from the 'hello-world-retro-2021' scripts, so I left them to err on caution. Let me know if we should also remove those!

Reply:
oh interesting I thought we removed them so I misremembered that! But definitely no need to clean them up right now